rasa/core/policies/mapping_policy.py

```diff
@@ -32,6 +32,23 @@ def __init__(self, priority: int = 3) -> None:
 
         super(MappingPolicy, self).__init__(priority=priority)
 
+    @staticmethod
+    def validate_against_domain(ensemble, domain):
```

Comment:
types

Reply:
Can't import `PolicyEnsemble` here as that'd cause a circular import